packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -309,20 +309,68 @@ export class ProfilesUtils {
         }
     }
 
+    /**
+     * Prompt whether to disable credential management setting
+     *
+     * This will disable credential management on all settings
+     * scopes since order presedence can be hard to predict based on the user's setup
+     */
+    public static async promptAndDisableCredentialManagement(): Promise<void> {
+        ZoweLogger.trace("ProfilesUtils.promptAndDisableCredentialManagement called.");
+        const noButton = localize("ProfileUtils.promptAndDisableCredentialManagement.yesButton", "No");
+        const yesButton = localize("ProfileUtils.promptAndDisableCredentialManagement.yesButton", "Yes");
+        const response = await Gui.warningMessage(
+            localize(
+                "ProfileUtils.promptAndDisableCredentialManagement.warning",
+                // eslint-disable-next-line max-len
+                "Zowe Explorer failed to activate since the default credential manager is not supported in your environment. Do you wish to disable credential management for all setting scopes? (VS Code window reload will be triggered)"
+            ),
+            {
+                items: [noButton, yesButton],
+                vsCodeOpts: {
+                    modal: true,
+                },
+            }
+        );
+        if (response === yesButton) {
+            await SettingsConfig.setDirectValueForAll("zowe.security.secureCredentialsEnabled", false);
```

Comment:
Curious why we set the setting across all scopes. Would it be sufficient to set it only in the currently active scope?

When I select "No", if I have a workspace open in VS Code that contains a `.vscode/settings.json` file (such as the ZE repo), then the settings.json file in the workspace is updated which results in uncommitted changes. Not sure if this is intended behavior.

Reply:
I have the same concern, we don't want to automatically change files that users push to the scm.

Reply:
Oops, I meant when selecting "Yes" 😆 When selecting "No", there are no changes to `settings.json` as expected 👍 

I can't think of a good way to automatically determine the currently active scope. Maybe we can leave the choice up to the user. What would you think of giving them a choice like the following?
<img width="269" alt="image" src="https://github.com/zowe/zowe-explorer-vscode/assets/22344007/34a26537-fce1-4ec7-ab6e-a91d9cff5349">